s3_logs/main.tf

```diff
@@ -1,8 +1,32 @@
+
+data "aws_iam_policy_document" "policy_document" {
```

Comment:
now this is defined at the s3_logs module level, when using this directly insteas of via the s3 module, users get the extra restrictions outlined in `default_log_bucket_policy.json.tpl` by default.

See https://github.com/nationalarchives/da-ayr-terraform/pull/237 for example